spec/models/article_spec.rb

```diff
@@ -613,5 +613,23 @@ def build_and_validate_article(*args)
     end
   end
 
+  describe ".feed" do
+    it "returns published articles ordered by most recent published_at" do
+      first_article = create(:article, published: true, published_at: 2.hours.ago)
+      second_article = create(:article, published: true, published_at: 1.hour.ago)
+      create(:article, published: false)
+
+      expect(described_class.feed.pluck(:id)).to eq([second_article.id, first_article.id])
+    end
```

Comment:
this is more a controller test, see the doubt about adding order to scopes

Reply:
I like to have query related tests in the model, but per other comment happy to move if desired.